robot/Cumulus/tests/browser/axe_test/first_axe_test.robot

```diff
@@ -0,0 +1,23 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Analyze account object home page
+    Inject Axe Core Library
```

Comment:
I see `Inject Axe Core Library` in a lot of places.  Is it possible to call this command in `Suite Setup` so `Inject Axe Core Library` is done once and the **Test Cases** don't need to worry about it?

Reply:
Similar comment, can the following commands be combined into a single command to make it a one-line to check for axe violations?

```robot
** combine these**
    &{acc_results}    Get Axe Analysis Results
    Log Summary Of Results    ${acc_results}
    Warn On Incomplete Rules    ${acc_results}
    Warn On Violations Rules    ${acc_results}

** so we can instead write something like **
    Expect No Axe Viloations ${element_to_test}
```